files/private-chef-cookbooks/private-chef/recipes/default.rb

```diff
@@ -156,6 +156,10 @@
   cert, key = OmnibusHelper.gen_certificate
 end
 
+# TODO: we are now using public_keys instead of certs and the
+#       bootstrap process now uses the public key. This is still
+#       here for backwards compatibility in case it is used anywhere
+#       else but we should consider deprecating it in the future.
```

Comment:
But I think we do need to reconfigure apps. oc_erchef should be loading the public key and not the certificate. We should delete the certificate file. We're just adding complexity. And now that I think about it, why can't erchef just load the public key from the database?